cloud/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.9-slim AS base
+
+LABEL "repository"="https://github.com/nitrictech/nitric"
+LABEL "homepage"="https://nitric.io"
+LABEL "maintainer"="Nitric Team <team@nitric.io>"
+LABEL org.opencontainers.image.description="Nitric common deployer, in a Docker container."
+
+# Install deps all in one step
+RUN apt-get update -y && \
+  apt-get install -y \
+  apt-transport-https \
+  build-essential \
+  ca-certificates \
+  curl \
+  git \
+  gnupg \
+  software-properties-common \
+  wget \
+  unzip && \
```

Comment:
If we use pulumi/pulumi-base we should have everything we need that they rely on.